.envrc

```diff
@@ -0,0 +1 @@
+use_nix
```

Comment:
should this file have been added…?

Reply:
actually yes. it does not disturb other tooling and should be part of the repository